Sources/UI/Cells/RequestCell.swift

```diff
@@ -10,20 +10,20 @@ import UIKit
 
 class RequestCell: UICollectionViewCell {
     
-    @IBOutlet weak var methodLabel: WHLabel!
     @IBOutlet weak var codeLabel: WHLabel!
-    @IBOutlet weak var urlLabel: WHLabel!
+    @IBOutlet weak var hostLabel: WHLabel!
+    @IBOutlet weak var pathLabel: WHLabel!
+    @IBOutlet weak var queryLabel: WHLabel!
     @IBOutlet weak var durationLabel: WHLabel!
     
     func populate(request: RequestModel?){
         guard request != nil else {
```

Comment:
We should replace this `guard` with a `guard let req = request`, and then avoid the force-unwrap and optional chaining in all the rest of the code (e.g. `let code = request.code` instead of `request?.code ?? 0`, `String(request.code)` instead of `String(request!.code)`, etc)